python/test/gherkin_test.py

```diff
@@ -89,3 +89,18 @@ def test_change_the_default_language():
     }
 
     assert expected == feature_file
+
+@pytest.mark.parametrize("trailing_text", ["\\", "\\ "])
+def test_inconsistent_cell_count_with_trailing_escape(trailing_text):
+    feature_text = """Feature:
+    Scenario:
+      Given I have a table
+        | Name | Value |
+        | A    | """ + trailing_text
+    parser = Parser()
+
+    with pytest.raises(
+        CompositeParserException,
+        match="inconsistent cell count within the table",
```

Comment:
I don't think this exception is quite right. In a table, a `\` must be followed by another character. So this is a different error. A regular party Box I guess.


```

        | Name | Value |
        | A    |  \| 
```

And this would be an inconsistent table.

Reply:
Ah. I see.  I missed:

https://github.com/cucumber/gherkin/blob/f5cbb5d7f52de872eb881526ea5eff4a980bbd7e/python/gherkin/gherkin_line.py#L66C1-L69

So this is the right error.